app/services/done_index_job.rb

```diff
@@ -0,0 +1,44 @@
+class DoneIndexJob < BaseIndexJob
+  attr_reader :status
+
+  def self.build_object(body)
+    new(status: body[:status],
+        book_version_id: body[:book_version_id],
+        indexing_version: body[:indexing_version])
+  end
+
+  def initialize(status:, **opts)
+    super(opts)
+    @status = status
+  end
+
+  def call
+    # add anything here that gets done when this done job is processed
+  end
+
+  def as_json(*)
+    {
+      "type" => @type,
+      "book_version_id" => @book_version_id,
+      "indexing_version" => @indexing_version,
+      "status" => @status
+    }
+  end
```

Comment:
ditto using `{x: y}` symbol hash notation and also the attr_reader methods available.